### ISSUE TITLE
Remove Sell, PowerDown and Beacon

### ISSUE THREAD

**Issue by Micr0Bit** [OpenRA/d2#5]
These didnt exist in the original

**Comment by Walkman100** [comment]
One of OpenRA's aims is not to exactly replicate the original games, but to add features from modern games to them - like the veterancy and right-clicking for certain actions instead of left-clicking for everything.

I think at least Sell and Beacon should be left in, PowerDown could be taken as cheating, idk.

**Comment by Micr0Bit** [comment]
@Walkman100 : i think you dont understand ... we are trying to create a "Dune2" ... you could no "sell" buildings ... "selling buildings" isnt even a new feature that got introduced by the "OpenRA-Engine" ...

this "Dune2" isnt OpenRA ... its a "modification" that is trying to recreate (maybe exactly) Dune 2 ... 

The OpenRA-Engine you are talking about ... lets you do "whatever" you want ... and doesnt have "any" restrictions towards any kind of ideas ... recreating , reimagine or whatever you wanna do with it ... you can do it ....

**Comment by Walkman100** [comment]
> Did you ever play Dune 2 actually ?

yes,  which is why I'm commenting here.
> now what Dune2 makes it special is ... that it doesnt have it

I see. I still think it'd be nice to have, but if this project's aim is to stick to the original as much as possible, then so be it.

**Comment by abcdefg30** [comment]
Beacons are user interaction... They don't have a direct impact on gameplay.
Removing selling and powerdown sounds good.

**Comment by Micr0Bit** [comment]
@abcdefg30 : agree ... otherwise , we would have to remove even the "chat" ... since that did also not exist ...fixing the title

**Comment by TheRaffy** [comment]
In the original Dune2 wasn't multiplayer. Will multiplayer now get deactivated?
In the original was only single selection of units possible. ,...
I mean the openra engine does not only provide the potential of a great **multiplyer** d2 mod.
Consider using the full potential of the engine for example weather effects. additional units like flasktruks, mobile deathhand, mtp's,.. additional terrain -> like sand to mountains tiles,.. Sure I understand your motivations, to create a good Dune2, but in my opinion I would vote for keeping:
-beacons, selling, powerdown, ranking (levelup freature),.. 
Will the balancing be like in the original d2? or like that from dune2 the golden path (rock-paper-scissors = foot-wheel-chain)? 
**The goals and vision of that mod should be discussed and written down.** 
When it comes to me, I'll support any additional feature and additional content, but I also appreciate the right of the active developer, so I don't blame anybody for not doing what I think would be the best.

**Comment by Micr0Bit** [comment]
@TheRaffy : our goal should be to create a vanilla-mod (as close as possible to the original) ... so that you or really anyone then can work from there and maybe create a "OpenRA-D2tgp" as offspring

just because OpenRA-engine provides us with tools that lets us add all the features and schnickschnack that "every other mod has,  doesnt mean that we need to add those 

Dune2000 for example does use everything that RA and TD offers ... and its a dead mod 
Dune2tgp does have a couple of features that new RTS-games have ... still the community is dead 

i believe that having a mod that somehow limits the player , challenges him ...  (and in that regard needs new gameplay-features) instead of adding the same old boring stuff from other mods we will benefit from the diversity

let alone drackbolts dune2-vision ... it was his own private project

**Comment by TheRaffy** [comment]
First priority, write down the vision statement/ goals of this mod. To avoid pulling in different directions. 
That minimizes conflict in the team and is damn helpful, when it comes to further decisions making.
Should there be a single person as visionkeeper? (if not make it an uneven count^^)

It is really not done by saying "vanilla mod + a maybe continue from there". 
Lol, dude a "maybe", a maybe is the totally opposite of what we need.

There are a lot open questions and things to discussion.
Believe me, if the vision is not nailed down, as precise as possible, this will end in a mess like d2k.

I guess this github is not the correct place, but better here than not at all.

- What does as close as possible mean in your kind of definition? 
 Does a "as close as possible" make scene? The closest you can get is the original. Why not playing the original instead? 
- When is your goal of "as close as possible" is reached?
 Do we need to insert 7 floppydisks`? and also install mousedrivers?, single move units around and playing with a tiny 320x Whatever screen resolution?
- I guess we can agree on that there's a thin line between different things that should stay original, things that need to change, and stuff that have to change. 
- **Who, besides the possibilities of the engine, decides where that line is?**

When it comes to me:
I like D2tgp a lot, as you know :) And I would love to see some adaptations, but get me, I'm not looking forward to just an "unbugged d2tgp + community". This mod could become get way better.
But I'm also not looking forward to a d2 mod as identical as possible. 
Sure I'm a little fangirl of that d2tgp, but i also like other Dunes like Super Dune for example.

I don't like to dive into details here. But it's a very bad excuse to say that possible enhancement is only allowed in d2000, like the only purpose of the d2k mod is to not need to enhancement d2.
It's like saying RA doesn't need enhancement, because there's an (inofficial) RA2 Mod. That's pure nonsense, sorry to say that.
And Creating a "OpenRA-D2tgp" as an offspring, while letting go d2tgp? That also doesn't make sense to me.

What is your hidden agenda?

**Comment by Micr0Bit** [comment]
-i want a userinterface that looks similar to Dune2 with a chatbar that ouputs information in realtime (for example how full (%) a harvester is) and i want the health of units displayed on a avatar of the selected unit near radar-section ... 

-i want single-selection instead of the lasso-grouping and Command-Buttons to give orders such as attack,stop,move,deploy .. and so on,  instead of the right-click for everything (the right-click-orders should get removed completely or suspended in a settings-option that is turned off by default

-i want a unit-cap of 25 "producible" units per player on default ... that can get raised in a settings-option up to 99 , nonetheless ... everyone can bypass whatever unit-cap by "ordering" units via starport 

-i want the original unit-balance , and when theres something unbalanced , then work it out gently by juggling with the unit-values 
i do "not" want any additional unit-assets to balance anything ... any foreign unit-assets or tileset-additions should be kept separated from the main-d2-mod but still an option for modders or any forks

-i want sell and powerdown out , this didnt exist in the original ... i see no reason why we would need it ... in fact , low "energy" didnt have any effect on production in the original ... so the windtrap was only important for the tech-tree ... but had no other use, i did read a post from MrFlibble on FED2k that there are people playing the Dune2 campaigns with only 1 Windtrap each game 

i would agree that this doesnt sound right , so here i would say an exception can be made and we add the "production-penalty" when on low power .... this should be discussed tho 
still, this doesnt justify a powerdown-button/action

Having no "sell" button makes it a bit more unforgiving on mistakes , but i dont see that as a big problem yet ... and you couldnt sell in the original 

-BuildOffAlliesConyard "OFF" on default
-First MCV should always start deployed
-Undeploy should be removed

**Comment by MustaphaTR** [comment]
>so the windtrap was only important for the tech-tree ... but had no other use

That's not correct. During lowpower, all buildings (even wind traps, which was annoying, you were continueing to lose power), even when on Concrete, slowly took damage to 50% max.

**Comment by Micr0Bit** [comment]
>That's not correct. During lowpower, all buildings (even wind traps, which was annoying, you were continueing to lose power), even when on Concrete, slowly took damage to 50% max.

Pardon me!

_Construction  yards  do  not  draw  power  from  windtraps.  However,  if  windtraps 
are  not  meeting  the power  demands  of  other  friendly  buildings,  then  all
friendly  buildings,  including  construction  yards, become susceptible to damage from the lack of power._

also ... 

_Without an outpost and/or sufficient power, the radar screen will only show friendly buildings._

**Comment by TheRaffy** [comment]
Thanks @Micr0Bit, now I kinda know what's your plan is. 
Does evgeniysergeev know where this mod is going? I know he has little spare time and likes that anybody continue his work, but to be fair I would love to consider his opinion, too. 

Was there any discussion meeting about that? or any pools?
@abcdefg30 , @Phrohdoh , @MustaphaTR , @evgeniysergeev , 
Do you guys care?

**Comment by MustaphaTR** [comment]
I prefer D2 mod to be as close to original as possible too. Most of those stuff are what makes D2 special.

**Comment by abcdefg30** [comment]
I also like the idea of getting this close to the original. (Unfortunately, ) I never played the original, so I have to trust you guys on that. ^^
(I.e. am fine with starting deployed, without being able to undeploy [d2k does it the same way btw], and even with removing power down [plan to do that for RA2 as well]. "Worst case" we can still add options to enable/disable those.)

**Comment by yaaaaa** [comment]
""Worst case" we can still add options to enable/disable those."
Yes, I think if this options will be on create game window in any mod, then engene de most flexible for players
